explorer/ast/bindings.h

```diff
@@ -78,9 +78,7 @@ class Bindings : public Printable<Bindings> {
   auto witnesses() const -> const ImplWitnessMap& { return witnesses_; }
 
   // Determine whether this is an empty set of bindings.
-  [[nodiscard]] auto empty() const -> bool {
```

Comment:
Optional, no change needed: I lean toward keeping this one, although I don't feel too strongly about it: "empty" is both an adjective and a verb, and people who naively read it as a verb are likely to disregard the return value, which is almost certainly a bug. On the other hand, experienced C++ programmers are unlikely to make that mistake, because of experience with `empty` methods on containers.

Reply:
Reverted on the basis that explorer is archived, so I'm not going to feel bad having it have readability issues.